components/automate-ui/src/app/pages/user-details/user-details.component.ts

```diff
@@ -45,56 +43,41 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
     private fb: FormBuilder
   ) {
     this.createForms(this.fb);
-    // TODO (tc) This needs to be refactored to resemble our other patterns
-    // for specific object pages.
-    // combineLatest depends on the user object existing already.
-    this.user = {
-      id: '',
-      name: '',
-      membership_id: ''
-    };
-
-    this.done$ = <Observable<boolean>>combineLatest([
```

Comment:
adding the store reset to the GET actions broke this page, so I refactored it to work a little more like team and token details.

Reply:
Excellent!